frontend/deploy/build.sh

```diff
@@ -1 +1,6 @@
 
+
```

Comment:
Adding more empty space will not be good

Reply:
Ok Will remove all the empty spaces in repo